app/javascript/components/Greeting.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const Greeeting = () => {
+  const greeting = useSelector((state) => state.greeting);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getGreetingAsync());
+  }, []);
+  return (
+    <div>
+      <h1>getGreetingAsync</h1>
+    </div>
+  );
+};
```

Comment:
- [ ] I can see you are retrieving the greeting data on line  4, but not rendering it to show on page did you forget that ? 😁  Kindly make sure that random greeting message is rendered.